dipy/reconst/tests/test_mapmri.py

```diff
@@ -29,7 +29,7 @@ def int_func(n):
     f = np.sqrt(2) * factorial(n) / float(((gamma(1 + n / 2.0)) *
                                           np.sqrt(2**(n + 1) * factorial(n))))
     return f
-
+s
```

Comment:
small typo mistake here and travis does not like it ;-)

Otherwise, Thanks for this update !

Reply:
Fixed. Thanks for the reveiw @skoudoro. Let's see what travis says about d9a44d2